src/Maui/Features/Creatures/Import/Create.razor.cs

```diff
@@ -0,0 +1,38 @@
+﻿using TabletopPowerTools.Core.Features.Creatures.Import;
+using TabletopPowerTools.Core.Models;
+
+namespace TabletopPowerTools.Maui.Features.Creatures.Import;
+public partial class Create
+{
+    [Inject] public IMediator Mediator { get; set; }
+    Creature creature = new();
+    string abilityName = "";
+    string abilityDescription = "";
+    string actionName = "";
+    string actionDescription = "";
+
+    public async void CreateCreature()
+    {
+        await Mediator.Send(new AcceptCreatureCommand { Creature = creature });
```

Comment:
When vertically slicing by features, features ideally shouldn’t share code to prevent a spaghetti mess of unintended consequences. Accepting a creature is for importing, not creation.

code should have as close to one reason to change as possible.